Brain/Brain.ino

```diff
@@ -40,9 +39,7 @@ sensorInfo all_ids[11] = {
    {"Load Cell Engine Right",     -1, -1, 10, 5, NULL}
 };
 
-sensorInfo sensor {"", 0, 0, 0, 0, 0};
-
-int numSensors = 11;
```

Comment:
we want this var

Reply:
Again why do we need numSensors I don't see it in use

Reply:
scratch that. my b